validator/src/bootstrap.rs

```diff
@@ -1205,7 +1204,7 @@ fn download_snapshot(
     download_abort_count: &mut u64,
     rpc_contact_info: &ContactInfo,
     desired_snapshot_hash: (Slot, Hash),
-    snapshot_kind: SnapshotKind,
+    archive_kind: ArchiveKind,
```

Comment:
I think we can leave the variable name, and then just use the new enum.

```suggestion
    snapshot_kind: SnapshotArchiveKind,
```